### PR TITLE
LPS-158167 Adjust label position

### PR DESCRIPTION
**Pull request by p2kmgcl**
Needs: `feature.flag.LPS-143206=true`
Checkout story definition for more info https://issues.liferay.com/browse/LPS-136279 :rainbow:

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/_SelectField.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	}
+
+	#{$prefix}:hover,
+	#{$prefix}:focus-within {
+		background-color: $cadmin-gray-100;
+	}
+
+	#{$prefix}:focus-within {
+		box-shadow: $btn-focus-box-shadow;
+	}
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/_SelectField.scss:37]: Very clean 👏

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js [hunk 2]
```diff
@@ -215,6 +229,126 @@ const SingleSelect = ({disabled, field, onValueSelect, options, value}) => {
...
+		);
+	}, [activeItemId, field.cssProperty, globalContext]);
+
+	return (
+		<div className="btn btn-unstyled p-0 page-editor__single-select-with-icon">
+			<label
+				className="mb-0 page-editor__single-select-with-icon__label-icon"
+				htmlFor={inputId}
+			>
+				<Tooltip
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js:311]: Do we need to use the custom tooltip here? can we use the liferay tooltip?

**Comment by victorg1991** [comment]
Talked with pablo and there are some minor things that needs update :P

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js (hunk 2 above)
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js]: Totally right, I just updated it :D

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**